docs/faststore/components/features/delivery-promise/implementation.mdx

```diff
@@ -26,8 +26,8 @@ deliveryPromise: {
   },
 ```
 
-* `enabled`: When set to **true**, Delivery Promise is activated, allowing the store to provide delivery estimates based on user location.
-* `mandatory`: When set to **false**, providing a postal code is optional for the user. Users can browse and add items to the cart without entering their postal code, though the accuracy of delivery information may be limited.
+* `enabled`: When set to **true**, Delivery Promise activates, allowing the store to provide delivery estimates based on the user's location.
+* `mandatory`: When set to **false**, providing a postal code is optional for the user. Users can browse and add items to the cart without entering their postal code, although the accuracy of delivery information may be limited.
```

Comment:
how about explaining what happens when set to **true**, and then explain the default? Something like:

```suggestion
* `mandatory`: When set to **true**, users must provide a postal code. By default, this is **false**, allowing users to browse and add items to the cart without entering their postal code. However, without a postal code the accuracy of delivery information may be limited.
```